bokehjs/src/coffee/source/ajax_data_source.coffee

```diff
@@ -48,17 +49,14 @@ class AjaxDataSource extends RemoteDataSource.Model
   defaults: =>
     return _.extend {}, super(), {
       mode: 'replace'
+      url : ""
+      expr : null
+      contentType : 'application/json'
+      headers : ""
```

Comment:
Actually another small question: is this the correct default? I would expect an empty dict/object